examples/simple-setup/src/index.js

```diff
@@ -1,14 +1,15 @@
 import "./styles.css";
 import embed from "./configure";
 import connect from "./connect";
-
+import { CONNECTION_TYPES } from "./connect";
```

Comment:
```suggestion
import connect, { CONNECTION_TYPES } from "./connect";
```
and remove the code one line above